docs/sources/operators-guide/configuring/configure-object-storage-backend.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Configuring Grafana Mimir object storage backend"
+menuTitle: "Configuring object storage"
+description: "Learn how to configure Grafana Mimir to use different object storage backend implementations."
+weight: 70
+---
+
+# Configuring Grafana Mimir object storage backend
+
+Grafana Mimir can use different object storage services to persist blocks containing the metrics data, as well as recording rules and alertmanager state.
+The supported alternatives are:
+
+- [Amazon S3](https://aws.amazon.com/s3/) (and compatible implementations like [MinIO](https://min.io/))
```

Comment:
This list is also referenced in `docs/sources/operators-guide/architecture/about-grafana-mimir-architecture/index.md` (see "Long-term storage"). Maybe we could remove it from there and keep it here? Don't feel strong, but just wanted you to consider it.

Reply:
I'd rather keep both places, I think this information is worth being at first glance in both places.